docs/getting-started-apm-server.asciidoc

```diff
@@ -0,0 +1,314 @@
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-x86_64.rpm
+sudo rpm -vi apm-server-{version}-x86_64.rpm
+----------------------------------------------------------------------
+
+endif::[]
+
+[[linux]]
+*Other Linux:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-linux-x86_64.tar.gz
+tar xzvf apm-server-{version}-linux-x86_64.tar.gz
+------------------------------------------------
+endif::[]
+
+[[mac]]
+*Mac:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O {downloads}/apm-server-{version}-darwin-x86_64.tar.gz
+tar xzvf apm-server-{version}-darwin-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[[installing-on-windows]]
+*Windows:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of APM Server has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+. Download the APM Server Windows zip file from the
+https://www.elastic.co/downloads/apm/apm-server[downloads page].
+
+. Extract the contents of the zip file into `C:\Program Files`.
+
+. Rename the `apm-server-<version>-windows` directory to `APM-Server`.
+
+. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon and select *Run As Administrator*).
+If you are running Windows XP, you may need to download and install PowerShell.
+
+. From the PowerShell prompt, run the following commands to install APM Server as a Windows service:
++
+[source,shell]
+----------------------------------------------------------------------
+PS > cd 'C:\Program Files\APM-Server'
+PS C:\Program Files\APM-Server> .\install-service-apm-server.ps1
+----------------------------------------------------------------------
+
+NOTE: If script execution is disabled on your system,
+you need to set the execution policy for the current session to allow the script to run.
+For example: `PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-apm-server.ps1`.
+
+endif::[]
+
+[[docker]]
+*Docker:*
+
+See <<running-on-docker, Running on Docker>> for deploying Docker containers.
+
+// *******************************************************
+// STEP 2
+// *******************************************************
+
+[[apm-server-configuration]]
+=== Step 2: Setup and configure
+
+[float]
+==== Elastic Cloud
```

Comment:
I suggest to use the same terminology as above [Hosted Elastic Service](https://github.com/elastic/apm-server/pull/2170/files#diff-231fc508dffaf5b3009bd25f92cc2dc7R12); same for the on-prem alternative.